modules/services/autoraise.nix

```diff
@@ -0,0 +1,160 @@
+            };
+            invertIgnoreApps = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Turns the ignoreApps parameter into an includeApps parameter. The default is false.";
+            };
+            ignoreApps = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of apps for which you would like to disable focus/raise.";
+            };
+            ignoreTitles = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of window titles (a title can be an ICU regular expression) for which you would like to disable focus/raise.";
+            };
+            stayFocusedBundleIds = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of app bundle identifiers that shouldn't lose focus even when hovering the mouse over another window.";
+            };
+            disableKey = lib.mkOption {
+              type = nullOr (enum [
+                "control"
+                "option"
+                "disabled"
+              ]);
+              default = null;
+              description = "Set to control, option or disabled. This will temporarily disable AutoRaise while holding the specified key. The default is control.";
+            };
+            mouseDelta = lib.mkOption {
+              type = nullOr float;
+              default = null;
+              description = "Requires the mouse to move a certain distance. 0.0 = most sensitive whereas higher values decrease sensitivity.";
+            };
+            verbose = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Set to true to make AutoRaise show a log of events when started in a terminal.";
+            };
+          };
+        };
+        default = { };
+        example = lib.literalExpression ''
+          {
+            pollMillis = 50;
+            delay = 1;
+            focusDelay = 0;
+            warpX = 0.5;
+            warpY = 0.1;
+            scale = 2.5;
+            altTaskSwitcher = false;
+            ignoreSpaceChanged = false;
+            invertIgnoreApps = false;
+            ignoreApps = [ "IntelliJ IDEA" "WebStorm" ];
+            ignoreTitles = [ "\\s\\| Microsoft Teams" ];
+            stayFocusedBundleIds = [ "com.apple.SecurityAgent" ];
+            disableKey = "control";
+            mouseDelta = 0.1;
+          }
+        '';
+        description = ''
+          AutoRaise configuration, see
+          <link xlink:href="https://github.com/sbmpost/AutoRaise"/>
+          for supported values.
+        '';
+      };
+    };
+  };
+
+  config = (
+    lib.mkIf (cfg.enable) {
+      environment.systemPackages = [ cfg.package ];
```

Comment:
Is this necessary? Running a second instance of AutoRaise is known to cause problems, so I'm not sure what the benefit of adding this to `systemPackages` is.

Reply:
I think we're not super consistent about it, but since AutoRaise doesn't have a man page, I think adding it has more cons than pros (especially since `autoraise --help` just runs it normally)